filewriter_v2/README.md

```diff
@@ -0,0 +1,388 @@
+| [detector_readout_time](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-detector-readout-time-field) | NX_FLOAT | Readout dead time between consecutive detector frames. |
+| detector_readout_time@units | NX_TIME | `"s"` |
+| [distance](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-distance-field) | NX_FLOAT | Distance from the sample to the beam center on the detector. |
+| distance@units | NX_LENGTH | `"m"` |
+| [flatfield_applied](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-flatfield-applied-field) | NX_BOOLEAN | Indicates whether a flatfield has been applied. |
+| [frame_time](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-frame-time-field) | NX_FLOAT | Time interval between start of image acquisitions. This defines the speed of data collection and is the inverse of the frame rate, the frequency of image acquisition. |
+| frame_time@units | NX_TIME | `"s"` |
+| [number_of_cycles](https://manual.nexusformat.org/classes/base_classes/NXdetector.html#nxdetector-number-of-cycles-field) | NX_INT | When autosummation is enabled, each frame is constructed by summing multple subframes. `number_of_cycles` is the number of summed subframes. If autosummation is disabled, this field has the value `1`. |
+| [pixel_mask_applied](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-pixel-mask-applied-field) | NX_BOOLEAN | Indicates whether a pixel mask has been applied. |
+| [saturation_value](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-saturation-value-field) | NX_INT | The value at which the detector goes into saturation. Data above this value is known to be invalid. |
+| [sensor_material](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-sensor-material-field) | NX_CHAR | Material used for direct detection of X-rays in the sensor. |
+| [sensor_thickness](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-sensor-thickness-field) | NX_FLOAT | Thickness of the sensor material. |
+| sensor_thickness@units | NX_LENGTH | `"m"` |
+| [serial_number](https://manual.nexusformat.org/classes/base_classes/NXdetector.html#nxdetector-serial-number-field) | NX_CHAR | Serial number of the detector. |
+| [start_time](https://manual.nexusformat.org/classes/base_classes/NXdetector.html#nxdetector-start-time-field) | NX_FLOAT[nP] | Relative start time for each frame, with `/entry/start_time` as absolute reference. |
+| [virtual_pixel_interpolation_applied](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-virtual-pixel-interpolation-applied-field) | NX_BOOLEAN | Indicates wheter virtual pixel interpolation has been applied. |
```

Comment:
whether

Reply:
fixed in [34890fd](https://github.com/soph-dec/documentation/pull/1/commits/34890fdc03430a5b054a735918d0593e38a8e157)